src/main/java/com/revealprecision/revealserver/config/LocationBatchConfiguration.java

```diff
@@ -73,7 +73,7 @@ public Job importLocationJob(LocationJobCompletionListener locationJobCompletion
   @Bean
   public Step step1() {
     return stepBuilderFactory.get("step1")
-        .<LocationRequest, Location>chunk(40)
+        .<LocationRequest, Location>chunk(100)
```

Comment:
Was this only for testing purposes or should it stay like this? Don't know if 100 is too much for chunk size..